ouroboros-network-framework/test/Test/Ouroboros/Network/Server2.hs

```diff
@@ -121,8 +121,10 @@ tests =
                  prop_connection_manager_pruning
   , testProperty "inbound_governor_pruning"
                  prop_inbound_governor_pruning
-  , testProperty "never_above_hardlimit"
-                 prop_never_above_hardlimit
+  {--
```

Comment:
Might be worth putting a TODO or a reason?

Reply:
Done, linked to the issue #3487.